transfers/manifestor.go

```diff
@@ -338,9 +348,13 @@ func (m *manifestorState) updateStatus(transferId uuid.UUID, entry manifestEntry
 		} else {
 			statusString = "failed"
 		}
-		pkg, err := datapackage.New(entry.Manifest, ".")
-		if err != nil {
-			return false, err
+		var pkg *datapackage.Package = nil
+		resources, ok := entry.Manifest["resources"].([]any)
+		if ok && len(resources) > 0 {
+			pkg, err = datapackage.New(entry.Manifest, ".")
+			if err != nil {
+				return false, err
+			}
```

Comment:
I wonder if we want to place the journal record and the message publishing inside the branch where `len(resources) > 0` and have a log message complaining about a zero-resource manifest in an `else` branch. What do you think of this?

Reply:
sounds good to me!